sherpa/ui/tests/test_ui_image.py

```diff
@@ -0,0 +1,693 @@
+#
+#  Copyright (C) 2021
+#  Smithsonian Astrophysical Observatory
+#
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License along
+#  with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+
+"""Basic tests of the miage functionality in sherpa.ui. The hope is that we
```

Comment:
```suggestion
"""Basic tests of the image functionality in sherpa.ui. The hope is that we
```

(I also love to fix really irrelevant typos in docstrings for tests!

Reply:
It is a spectacularly-good typo.